context/app/static/js/components/home/ExternalLinks/ExternalLinks.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { PropsWithChildren } from 'react';
+import Paper from '@mui/material/Paper';
+
+import ExternalLink from 'js/components/home/ExternalLink';
+import { Box, Stack } from '@mui/material';
+import { SectionHeader } from 'js/pages/Home/style';
+
+function RelatedResources() {
+  return (
+    <Paper>
+      <ExternalLink
+        img={<img src={`${CDN_URL}/v2/hubmap_consortium.png`} height={56} width={56} alt="HuBMAP Logo" />}
+        linkHref="https://hubmapconsortium.org/"
+        linkText="HuBMAP Consortium"
+        description="Navigate to the HuBMAP Consortium where members are developing the tools to create an open, global atlas of the human body at the cellular level."
+      />
+      <ExternalLink
+        img={<img src={`${CDN_URL}/v2/nih_common_fund.png`} height={56} width={56} alt="NIH Logo" />}
+        linkHref="https://commonfund.nih.gov/hubmap/"
+        linkText="Common Fund"
+        description="Learn about the NIH Common Fund, a funding entity that supports scientific programs that catalyze discovery across biomedical and behavioral research."
+      />
+      <ExternalLink
+        img={<img src={`${CDN_URL}/v2/protocols.io.png`} height={56} width={56} alt="Protocols.io Logo" />}
+        linkHref="https://www.protocols.io/workspaces/human-biomolecular-atlas-program-hubmap-method-development/"
+        linkText="HuBMAP Protocols"
+        description="View the HuBMAP Consortium’s protocols in protocols.io."
+      />
+      <ExternalLink
+        img={<img src={`${CDN_URL}/v2/google_scholar.png`} height={56} width={56} alt="Google Scholar Logo" />}
+        linkHref="https://scholar.google.com/citations?user=CtGSN80AAAAJ"
+        linkText="HuBMAP Publications"
+        description="Explore the list of related publications generated by members of the HuBMAP Consortium on their Google Scholar page."
+      />
+    </Paper>
+  );
+}
+
+function RelatedTools() {
+  return (
+    <Paper>
+      <ExternalLink
+        img={<img src={`${CDN_URL}/v2/hra_icon.png`} height={56} width={56} alt="HuBMAP Logo" />}
```

Comment:
Some repeated logic here. Seems like we can pull out the props into an obj to map over? Also they all have an alt of 'HuBMAP Logo'.

Reply:
Done 👍🏻 Thanks for catching the alt text issue 😆